renderdoc/android/android.cpp

```diff
@@ -31,6 +31,14 @@
 
 namespace Android
 {
+bool isFirstTargetControlPortUsed(const std::string &deviceID)
+{
+  string command =
+      StringFormat::Fmt("shell netstat -l | grep @renderdoc_%u", RenderDoc_FirstTargetControlPort);
```

Comment:
Using a new command is not justifiable in this case. Because Android is a terrible and fundamentally broken operating system there is no guarantee that this command will be available on all devices, or if it is available that it will accept the same parameters or output in the same format. In this case that would cause all android apps to be unlaunchable, which would be very serious.

Instead this check should happen in the UI itself, enumerating if any available target control connections are running on the device. If they are then the error can be displayed.

Reply:
I have modified my commit.